rplugin/python3/deoplete/sources/deoplete_jedi.py

```diff
@@ -192,11 +192,17 @@ def gather_candidates(self, context):
         if cached is None:
             wait = True
 
+        # Extra options to pass to the server.
+        options = {
+            'cwd': context.get('cwd'),
+            'runtimepath': context.get('runtimepath'),
```

Comment:
> Sending the whole &rtp is rather huge

This is part of the reason I'm not sending the entire `context` (re: above) since there's a lot more in `context` than just `&rtp`.  I prefer to be deliberate in what's sent to the server so it's minimal.  I also think this is a decent way to send arbitrary data to the server, which should've been done in the first place instead of using a fixed tuple.

I actually thought about sending `&rtp` as part of the server startup, but it would mean needing to restart if the paths change.